code/frameworks/svelte-vite/src/preset.ts

```diff
@@ -9,6 +9,8 @@ export const core: StorybookConfig['core'] = {
 
 export const viteFinal: NonNullable<StorybookConfig['viteFinal']> = async (config, options) => {
   const { plugins = [] } = config;
+  // TODO: set up eslint import to use typescript resolver
+  // eslint-disable-next-line import/no-unresolved
```

Comment:
This is a bit irritating.  ESLint started to fail here, I think because the new `@sveltejs/vite-plugin-svelte` is esm-only and uses `exports` with no `main` field in its `package.json`.  

I've opened https://github.com/sveltejs/vite-plugin-svelte/issues/552 since I think this is a mistake on the part of the plugin.